irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -142,6 +142,11 @@ namespace iroha {
       if (boost::size(transaction->signatures()) < transaction->quorum()) {
         log_->info("waiting for quorum signatures");
         mst_processor_->propagateTransaction(transaction);
+        status_bus_->publish(
```

Comment:
This status publishing is done in such way throughout this file, and I think it will be unnecessary hard to move it into a separate function, because we need to put the status somehow, and it's done through the builder. In case of separate function we can pass the builder, but why?

Reply:
So, after a little discuss, I decided that this task actually is about creating something like status factory, which should be done outside this PR

Reply:
Offline discussion:
Move status_bus::publish to the separate method and add a task for refactoring this method in a separate PR

Reply:
Offline discussion:
Move status_bus::publish to the separate method and add a task for refactoring this method in a separate PR